flexget/tests/test_sort_by.py

```diff
@@ -96,9 +96,9 @@ class TestSortBy:
               dict_field: {a: 2, b: 0}
     """
 
-    def generate_test_ids(param):
-        if param[0:5] == 'test_' or not isinstance(param, list):
-            return param
+    def generate_test_ids(self):
```

Comment:
I think this one is also a `@staticmethod`

Reply:
The modification failed the tests on Python 3.9, so I added `#noqa`. After dropping support for Python 3.9 later this year, `#noqa` can be removed and `@staticmethod` can be added.